path.go

```diff
@@ -3,22 +3,31 @@ package txtdirect
 import (
 	"context"
 	"fmt"
+	"log"
 	"net"
 	"regexp"
 	"sort"
 	"strconv"
 	"strings"
+	"time"
 )
 
 var PathRegex = regexp.MustCompile("\\/([A-Za-z0-9-._~!$'()*+,;=:@]+)")
 var FromRegex = regexp.MustCompile("\\/\\$(\\d+)")
 
 func zoneFromPath(host string, path string, rec record) (string, int, error) {
 	pathSubmatchs := PathRegex.FindAllStringSubmatch(path, -1)
+	if rec.Re != "" {
+		CustomRegex := regexp.MustCompile(rec.Re)
+		pathSubmatchs = CustomRegex.FindAllStringSubmatch(path, -1)
```

Comment:
My thought would be to prohibit using both and redirect the request to the default URL, so that the user realizes quickly something's wrong with his config.

Reply:
We can check if ``from=`` and ``re=`` are both available on the record before going into zoneFromPath and then run the fallback code.

Reply:
Yeah let's fail hard/strict here and iterate with actual user feedback later. So if both are present. Log an error and fallback.